lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1116,6 +1116,47 @@ LLVMToSPIRV::getLoopControl(const BranchInst *Branch,
   return static_cast<spv::LoopControlMask>(LoopControl);
 }
 
+static int transAtomicOrdering(llvm::AtomicOrdering Ordering) {
+  switch (Ordering) {
+  case AtomicOrdering::Monotonic: /* aka "relaxed" */
+    return MemorySemanticsMaskNone;
+  case AtomicOrdering::Acquire:
+    return MemorySemanticsAcquireMask;
+  case AtomicOrdering::Release:
+    return MemorySemanticsReleaseMask;
+  case AtomicOrdering::AcquireRelease:
+    return MemorySemanticsAcquireReleaseMask;
+  case AtomicOrdering::SequentiallyConsistent:
+    return MemorySemanticsSequentiallyConsistentMask;
+  default:
+    llvm_unreachable("Unsupported atomic ordering.");
+  }
+  return -1;
```

Comment:
Can we reuse these [2 lines](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/a825cdba21306daf40725107d354a6858aed638d/lib/SPIRV/SPIRVWriter.cpp#L1549-L1550) instead?

Reply:
Done.